polyfill/lib/ecmascript.mjs

```diff
@@ -870,6 +870,14 @@ export const ES = ObjectAssign({}, ES2019, {
       })
       .filter((x) => x !== undefined);
   },
+
+  ChooseCommonCalendar: (cal1, cal2) => {
+    if (cal1.id === cal2.id) return cal1;
+    if (cal1.id === 'iso8601' || cal1.id === 'gregory') return cal2;
```

Comment:
Here and elsewhere: I consider `"gregory"` to be stronger than `"iso8601"`.  I see `"gregory"` as an explicit opted-in Gregorian calendar for humans, whereas `"iso8601"` is an implicit non-human calendar.

Reply:
That was be my intuition as well. I did it this way, though, because it seems that Intl doesn't agree; `gregory` and not `iso8601` is the locale's calendar in `resolvedOptions` for locales that use the Gregorian calendar such as `en-CA`. Although, now that I'm looking back to #262, maybe that is correct nonetheless?

Reply:
Intl returns `"gregory"` because that's what CLDR defines as the default calendar system for locales such as `en-CA`.  It's a human calendar.  To the best of my knowledge, `"iso8601"` isn't covered in CLDR.  It's a computer calendar.  The ISO calendar happens to correspond very closely to the Gregorian calendar, but I think we need not equate the two; it seems useful for `"gregory"` to mean that you've explicitly opted in to the Gregorian calendar.